### ISSUE TITLE
Fix killSession()

### ISSUE THREAD

**Issue by pedrouid** [WalletConnect/walletconnect-monorepo#52]
Throws following error when called
```
Uncaught TypeError: First argument must be a string, Buffer, ArrayBuffer, Array, or array-like object. walletconnect.js:15
```

**Comment by pedrouid** [comment]
Looks like it comes from line 84 from walletconnect/src/index.js
https://github.com/WalletConnect/walletconnect-monorepo/blob/master/packages/walletconnect/src/index.js#L84
That calls `symKey` setter which includes a Buffer instantiation

**Comment by pedrouid** [comment]
Fixed on `v0.7.27`
